src/Three.TSL.js

```diff
@@ -24,6 +24,7 @@ export const PCFShadowFilter = TSL.PCFShadowFilter;
 export const PCFSoftShadowFilter = TSL.PCFSoftShadowFilter;
 export const PI = TSL.PI;
 export const PI2 = TSL.PI2;
+export const PI_HALF = TSL.PI_HALF;
```

Comment:
`HALF_PI` maybe?

Reply:
`PI05` is interesting... I'll let @sunag decide.

Reply:
Updated the PR to the new names.